Misc/NEWS.d/next/Library/2023-07-09-00-36-33.gh-issue-106558.Zqsj6F.rst

```diff
@@ -0,0 +1,3 @@
+Fix ref cycle in callers of
+:func:`~multiprocessing.managers.convert_to_error` by deleting ``result``
+from scope in a ``finally`` block if it's an exception.
```

Comment:
```suggestion
from scope in a ``finally`` block.
```

I think you're deleting it regardless of its type, right?

Reply:
Yeah—so long as it actually reached calling `convert_to_error` at all, we'll delete `result` with no further conditions on `kind`.